traffic_router/connector/src/main/java/com/comcast/cdn/traffic_control/traffic_router/protocol/RouterNioEndpoint.java

```diff
@@ -53,10 +56,10 @@ protected void initialiseSsl() throws Exception {
 		}
 	}
 
-	@SuppressWarnings({"PMD.NPathComplexity", "PMD.UseStringBufferForStringAppends"})
-	synchronized public void replaceSSLHosts(final Map<String, HandshakeData> sslHostsData) {
-		final Set<String> aliases = sslHostsData.keySet();
-		String lastHostName = "";
+    synchronized private List<String> replaceSSLHosts(final Map<String, HandshakeData> sslHostsData) {
```

Comment:
Do you mind reformatting the additions here? This file appears to be mostly tab-indented rather than space-indented.

Reply:
Done!